.github/workflows/clang-format-pull-request.yml

```diff
@@ -4,23 +4,22 @@ on:
   pull_request:
     branches:
       - release_oneDPL
-      - master
+      - main
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
       with:
-        fetch-depth: 1
+        fetch-depth: 0
 
     - name: Get clang-format
       run: sudo apt-get install -yqq clang-format-6.0
 
     - name: Applying clang-format for changed files
       run: |
-        URL="https://api.github.com/repos/${GITHUB_REPOSITORY}/pulls/${{ github.event.pull_request.number }}/files"
-        FILES=$(curl -s --request GET --get $URL |  jq -r '.[] | .filename' | grep ^include | grep -v nanorange\.hpp\$ || true)
-        echo $FILES | xargs -n1 -t -r clang-format-6.0 --style=file -i
+        FILES=$(git diff --name-only ${{ github.event.pull_request.base.sha }} | grep ^include | grep -v nanorange\.hpp\$ || true)
+        clang-format-6.0 -i -style=file $FILES
```

Comment:
I'd prefer to keep the previous approach here with `echo $FILES | xargs ...` since it handles empty `$FILES` more properly.